src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -444,7 +444,7 @@ def __read_kv_from_app_service(cmd, appservice_account, prefix_to_add="", conten
                             # this throws an exception for invalid format of secret identifier
                             parse_key_vault_id(source_id=secret_identifier)
                             kv = KeyValue(key=key,
-                                          value=json.dumps({"uri": secret_identifier}, ensure_ascii=False, separators=(',', ':')),
+                                          value=json.dumps({"uri": secret_identifier}, ensure_ascii=False),
```

Comment:
May I ask if these separators changes will lead to a breaking change for customer automation scripts?

Reply:
This change should not affect user automation scripts. The initial separators were added to remove whitespaces after the "," and ":" in a JSON serialized string. This removal of whitespace is not needed anymore and will not break any customer scenarios since the contents of the strings will still remain, and be used or written as parsed JSON values.